<launch>
    <include file="$(find amigo_machine_files)/machines/amigo4.machine" />

    <node pkg="move_arm" type="move_arm_simple_action" name="move_left_arm" output="screen" machine="amigo4">
        <param name="group" type="string" value="left_arm" />
        <param name="ik_allowed_time" type="double" value="2.0" />
        <param name="trajectory_filter_allowed_time" type="double" value="2.0" />       
        <param name="controller_action_name" type="string" value="/joint_trajectory_action_left" />

        <!--<remap from="move_left_arm/goal" to="move_left_arm/pre_grasp/goal" />-->
	    <remap from="joint_state" to="joint_states" />
        <remap from="arm_ik" to="amigo_left_arm_kinematics/get_constraint_aware_ik" />
        <remap from="arm_fk" to="amigo_left_arm_kinematics/get_fk" />
        <remap from="filter_trajectory" to="trajectory_filter/filter_trajectory_with_constraints" />
        <remap from="get_trajectory_validity" to="environment_server/get_trajectory_validity" />	
    	<remap from="get_environment_safety" to="environment_server/get_environment_safety" />	
    	<remap from="get_execution_safety" to="environment_server/get_execution_safety" />	
    	<remap from="get_group_info" to="environment_server/get_group_info" />	
    	<remap from="get_robot_state" to="environment_server/get_robot_state" />	
    	<remap from="get_state_validity" to="environment_server/get_state_validity" />
    </node>
</launch>
